### DISCUSSION TITLE
Fires

### DISCUSSION THREAD

**Discussion by Josiahisnerd** [ldtteam/minecolonies#7737] [Q&A] [answered]
I cannot find any info anywhere else, so here is the best place I can think of asking. My buildings are burning down specifically, the dark oak university and houses, do you know why this is?

**Comment by Justen-N** [comment, marked as answer]
The absolute best place to ask questions is the Discord: https://discord.gg/xp53SY6D 
Github is not set up for direct support.

**Reply by Roses4664** [reply]
Then why is this page called "Q&A" if you want us to ask questions and get answers to those questions on discord?
